src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+    private String description;
```

Comment:
Your use private variables and access functions instead of public variables adheres to Java's information hiding and encapsulation concepts. Good Job!